app-doc/heirloom-doctools/heirloom-doctools-160308.ebuild

```diff
@@ -0,0 +1,215 @@
+		vgrind/Makefile.mk || die
+	sed -i -e '/\/vgrindefs/s:LIBDIR:"/usr/share/heirloom/doctools":' \
+		vgrind/vfontedpr.c || die
+
+	sed -i -e 's:$(REFDIR)/papers:/usr/share/heirloom/reftools/papers:g' \
+		refer/Makefile.mk || die
+	sed -i -e '/\/papers\/Ind/s:REFDIR:"/usr/share/heirloom/reftools":' \
+		refer/refer1.c || die
+
+	# Correct paths for the installed man pages, just to be clean
+	find . \( -name '*.1' -or -name '*.1b' -or -name '*.7' -or -name '*.7b' \) -exec \
+		sed -i \
+			-e "s:/usr/ucblib/grap.defines:/usr/share/heirloom/doctools/grap.defines:" \
+			-e "s:/usr/ucblib/vgrindefs:/usr/share/heirloom/doctools/vgrindefs:" \
+			-e "s:/usr/ucblib/vfontedpr:/usr/libexec/heirloom/doctools/vfontedpr:" \
+			-e "s:/usr/ucblib/tmac/vgrind:/usr/share/heirloom/tmac/vgrind:" \
+			-e "s:/usr/ucblib/eign:/usr/share/heirloom/doctools/eign:" \
+			-e "s:/usr/ucb/:/usr/bin/:g" \
+			-e "s:/usr/ucblib/doctools:/usr/share/heirloom/doctools:g" \
+			-e "s:/usr/ucblib/reftools/papers:/usr/share/heirloom/reftools/papers:" \
+			-e "s:/usr/ucblib/reftools:/usr/libexec/heirloom/reftools:g" \
+		{} +
+
+	eapply_user
+}
+
+src_configure() {
+	append-cppflags -D_GNU_SOURCE
+
+	sed \
+		-e "s:@CFLAGS@:${CFLAGS}:" \
+		-e "s:@CXXFLAGS@:${CXXFLAGS}:" \
+		-e "s:@CPPFLAGS@:${CPPFLAGS}:" \
+		-e "s:@LDFLAGS@:${LDFLAGS}:" \
+		-e "s:@CC@:$(tc-getCC):" \
+		-e "s:@CXX@:$(tc-getCXX):" \
+		-e "s:@RANLIB@:$(tc-getRANLIB):" \
+		-e "s:@libdir@:$(get_libdir):" \
+		"${FILESDIR}"/${PV}.config \
+		> "${S}"/mk.config
+}
+
+src_compile() {
+	emake $(use cxx && echo MPM=mpm) \
+  BINDIR="/usr/bin" \
+	LIBDIR="/usr/share/heirloom/doctools" \
+  PUBDIR="/usr/share/heirloom/doctools/pub" \
+	MANDIR="/usr/share/man" \
+	MACDIR="/usr/share/heirloom/doctools/tmac" \
+	FNTDIR="/usr/share/heirloom/doctools/font" \
+	PSTDIR="/usr/share/heirloom/doctools/font/devpost/postscript" \
+	TABDIR="/usr/share/heirloom/doctools/nterm" \
+	HYPDIR="/usr/share/heirloom/doctools/hyphen" \
+	REFDIR="/usr/share/heirloom/reftools" \
+	|| die
+}
+
+src_install() {
+	dodir /usr/share/heirloom/{doc,ref}tools
+
+	# The build system uses the ROOT variable in place of DESTIDR.
+	emake $(use cxx && echo MPM=mpm) INSTALL="/usr/bin/install" STRIP="/usr/bin/strip" \
+  BINDIR="/usr/bin" \
+	LIBDIR="/usr/share/heirloom/doctools" \
+	PUBDIR="/usr/share/heirloom/doctools/pub" \
+	MANDIR="/usr/share/man" \
+	MACDIR="/usr/share/heirloom/doctools/tmac" \
+	FNTDIR="/usr/share/heirloom/doctools/font" \
+	PSTDIR="/usr/share/heirloom/doctools/font/devpost/postscript" \
+	TABDIR="/usr/share/heirloom/doctools/nterm" \
+	HYPDIR="/usr/share/heirloom/doctools/hyphen" \
+	REFDIR="/usr/share/heirloom/reftools" \
+	ROOT="${D}" install || die
+
+	dodoc README CHANGES || die
+
+	# Rename ptx to avoid a collision with coreutils… maybe this
+	# should be made conditional to userland_GNU (somebody got to
+	# check on FreeBSD).
+	mv "${D}"usr/bin/{,hl-}ptx || die
+	mv "${D}"/usr/share/man/man1/{,hl-}ptx.1 || die
```

Comment:
You are using `${D}usr` and `${D}/usr` inconsistently. Either use the former or `${D%/}/usr` since D ends with /.